ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -547,9 +548,11 @@ public static string method_toString()
                 "Comparisons|peak_width_base_et\t" + peak_width_base_et.ToString(),
                 "Comparisons|min_peak_count_ee\t" + min_peak_count_ee.ToString(),
                 "Comparisons|min_peak_count_et\t" + min_peak_count_et.ToString(),
-                "Families|family_build_folder_path\t" + family_build_folder_path,
-                "Families|deltaM_edge_display_rounding\t" + deltaM_edge_display_rounding.ToString()
+                //this doesn't seem like it makes sense to be a part of load and run -LVS
```

Comment:
It makes it nicer for making proteoform families, where you don't have to reselect a folder every time you want to open up results and construct your families differently.